tests/system/test_error_details.py

```diff
@@ -47,14 +45,21 @@ def create_bad_request_details():
         field_violation.field = "test field"
         field_violation.description = "test description"
         return bad_request_details
+
     bad_request_details = create_bad_request_details()
     status = create_status(bad_request_details)
 
     with pytest.raises(exceptions.GoogleAPICallError) as e:
-        _ = echo.echo(showcase.EchoRequest(
-            error=status,
-        ))
-        assert e.details == [bad_request_details]
+        _ = echo.echo(
+            showcase.EchoRequest(
+                error=status,
+            )
+        )
+
+    # Note: gRPC errors expose e.value.details.
+    exc_details = e.details if "rest" in str(
+        echo.transport) else e.value.details
+    assert exc_details == [bad_request_details]
```

Comment:
I've removed the variable and used `e.value.details` directly since it's the same for both rest and gRPC. We'll keep the tests disable for rest until the issue mentioned in the TODO is fixed.

Reply:
ok